test/Dir/DriveAbsPath/SConstruct

```diff
@@ -0,0 +1,32 @@
+import os
+import SCons
+
+env = Environment()
+drive = os.path.splitdrive(os.getcwd())[0]
+drive_dir = env.fs.Dir(drive)
+
+if not isinstance(drive_dir, SCons.Node.FS.RootDir):
+	raise Exception('env.fs.Dir("%s") returned a %s instance of a RootDir' %
```

Comment:
Shouldn't this be "not an instance of RootDir" ?

Reply:
Oops, I meant to put "instead" but ended up with "instance". Fixed.